src/walletSdk/KeyManager/plugins/IdentityEncrypter.ts

```diff
@@ -0,0 +1,39 @@
+import { DecryptParams, Encrypter, EncryptParams } from "../../Types";
+
+const NAME = "IdentityEncrypter";
+
+/**
+ * "Encrypt" keys in a very basic, naive way.
+ */
+export const IdentityEncrypter: Encrypter = {
+  name: NAME,
+  encryptKey(params: EncryptParams) {
+    const { key } = params;
+    const { type, privateKey, publicKey, path, extra, ...props } = key;
+
+    return Promise.resolve({
+      ...props,
+      encryptedBlob: JSON.stringify({
+        type,
+        publicKey,
+        privateKey,
+        path,
+        extra,
+      }),
+      encrypterName: NAME,
+      salt: "identity",
+    });
+  },
+
+  decryptKey(params: DecryptParams) {
+    const { encryptedKey } = params;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { encrypterName, salt, encryptedBlob, ...props } =
```

Comment:
@CassioMG the encrypterName and salt are being pulled out of props here so it's not returned below

do you know if that's on purpose? seems odd to me, since they're not being used

Reply:
It seems `encrypterName` and `salt` should be part of an `encrypted key` but not a `decrypted key`, so it's probably why they're being removed here to avoid adding extra unnecessary info to a `decrypted key`.

I've inferred it through `IdentityEncrypter's test` [here](https://github.com/stellar/js-stellar-wallets/blob/master/src/plugins/IdentityEncrypter.test.ts#L35), and we can also infer it from the return types of  `decryptKey` and  `encryptKey` declarations [here](https://github.com/stellar/js-stellar-wallets/blob/master/src/types/keys.ts#L84).

Reply:
gotchya, makes sense thanks!